src/pages/user/UserAffiliations.tsx

```diff
@@ -95,7 +95,7 @@ const UserInstitution = ({ user }: UserInstituionProps) => {
   return (
     <>
       <Card>
-        <Typography variant="h5">{t('heading.organizations')}</Typography>
+        <Typography variant="h5">{t('heading.affiliations')}</Typography>
```

Comment:
her skulle vi sikkert hatt en `variantMapping`. Tviler på at vi har både h1, h2, h3, og h4 her?

Reply:
var like greit å bare sette h2 direkte

Reply:
Dropper boksen for forfatteridentitet siden den skal bort ganske straks: https://unit.atlassian.net/browse/NP-2232